README.md

```diff
@@ -329,6 +330,12 @@ if you're in Chinatown.
 
 ![Wanton Min](/images/wanton_min.jpg)
 
+## Indie Bookstore
+
+- [BooksActually][booksactually]
+
+[booksactually]: https://www.booksactuallyshop.com/pages/about
+
```

Comment:
🤔 Wondering why not just `[]()` instead of `[][] []:xxx`. 🤔

Reply:
habit from other projects 😅 😅 

can keep text looks less unclutter, probably does not matter here, changed 👍 👍 👍